### PR TITLE
feat: add toggle to allow news and ideas to be archived

### LINKED ISSUE #170: Provide a way to unpublish content for archiving purposes
## Is your feature request related to a problem? Please describe.

It may be useful to provide a way to unpublish content and keep the content for archival purposes.

Currently when you unpublish content, it will appear in the Workflow with "Ready", "In Review", or "Draft" status which can clutter the interface from pages that are actively being worked on.

## Describe the solution you'd like

Add a 4th workflow status "Hidden", "Archived" or "Unpublished Archive".

## Additional context

Encountered this issue when cleaning up the staff pages and unpublished staff who have moved on from the IDRC.

### PR DESCRIPTION
**Pull request by greatislander**
Resolves #170.

* [x] This pull request has been tested by running `npm run test` without errors
* [x] This pull request has been built by running `npm run build` without errors
* [x] This isn't a duplicate of an existing pull request

## Description

As identified in #170, it can be beneficial to mark news or ideas posts as "archived" so that they can remain in source control but be hidden from the public site. This PR adds a switch to the CMS which lets content editors mark news ore ideas posts as "archived", and if they are marked as such they will be excluded from the build.

## Steps to test

1. Run `npm start`.
2. Visit the CMS at `http://localhost:8080/admin`.
3. Toggle the "Archived" switch for a news or ideas post, and the publish the changes.

**Expected behavior:** The news or ideas post should no longer be accessible on the local site.

## Additional information

Not applicable.

## Related issues

Once this is live, https://github.com/inclusive-design/idrc/pull/356 and https://github.com/inclusive-design/idrc/pull/382 can be modified to mark these two items as archived rather than leaving them as drafts, and then the PRs can be merged.

### CODE REVIEW

File: src/ideas/until-school-restarts.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: " Until School Restarts: Organize an “Ageless Kids’ Club”"
+date: 2020-03-29
+archived: true
+author: Jutta Treviranus
+thumbnailImage: /media/school-closed.jpg
```
Comment (cindyli) [inline src/ideas/until-school-restarts.md:6]: This image is missing from `src/media/` directory which causes the build to fail when turning off the "archive" flag.

Comment (greatislander) [reply src/ideas/until-school-restarts.md:6]: @cindyli yes, I'm not sure what to do about that as the image was also missing from this PR: #356. Should I delete the `thumbnailImage` attribute altogether, do you think? I did wonder if it was worth trying to find the original, perhaps on Jutta's Medium blog.

Comment (greatislander) [reply src/ideas/until-school-restarts.md:6]: Found it! [656cdd4](https://github.com/inclusive-design/idrc/pull/635/commits/656cdd498bb60d8b8a63d4128711e5ce35b4e3d7)

Comment (cindyli) [reply src/ideas/until-school-restarts.md:6]: @greatislander I agree it's not worth finding the original image. Deleting the `thumbnailImage` attribute is a good workaround. If you do so, please also delete the line 8 where the same image is refered in the article content.

Comment (cindyli) [reply src/ideas/until-school-restarts.md:6]: ah ha, just noticed you've restored the image. Cool.

Comment (greatislander) [reply src/ideas/until-school-restarts.md:6]: Thanks to the Internet Archive :)

**Review by cindyli — APPROVED** [review]

**Pull request merged**